sycl/test-e2e/Matrix/element_wise_all_ops_half.cpp

```diff
@@ -1,24 +0,0 @@
-//==----------- element_wise_all_ops_half.cpp  - DPC++ joint_matrix---------==//
-//
-// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
-// See https://llvm.org/LICENSE.txt for license information.
-// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
-//
-//===----------------------------------------------------------------------===//
-// REQUIRES: aspect-fp16
```

Comment:
Half variant of the test was not running on CPU, I think, because half is not supported on CPU?
In new implementation this condition disappeared, so the test will fail on CPU, right?
If so, I think we still may want to have half to be in separate cpp file, so that the rest of the tests (float, bfloat16) are passing on CPU and half is passing only on GPU.

Reply:
You are right, half test case `requires gpu`. I will keep half separated then.